src/components/TransactionItem/TransactionItem.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { StyledListItem } from "./TransactionItem.styled";
+
+const TransactionItem = (transaction) => {
```

Comment:
can we destruct this transaction, if you need to check how transaction looks like you have to debug, scan through code, write console log or something else, add prop types also

Reply:
so those are props, change name to props, one naming convention is important

Reply:
done